### PR TITLE
Issue 188: st2rulesengine references datastore_crypto_key incorrectly

### PR DESCRIPTION
**Pull request by lordpengwin**
I fixed the secret name for the rules engine

Closes #190

### CODE REVIEW

**Review by armab — CHANGES_REQUESTED** [review]
Looks good! Thanks for the contribution :+1: 

Could you also include a respective Changelog record for this PR?

**Comment by lordpengwin** [comment]
> Could you also include a respective Changelog record for this PR?

I'd be happy to but I'm not sure what you mean. Can you point me to an example of what you want?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## In Development
 * Change st2packs definition to a list, to support multiple st2packs containers (#166) (by @moonrail)
 * Enabled RBAC/LDAP configuration for OSS version, removed enterprise flags (#182) (by @hnanchahal)
+* Fixed datastore_crypto_key secret name for rules engine (#188) (by @lordpengwin)
```
Comment (armab) [inline CHANGELOG.md:6]: :+1:

Comment (lordpengwin) [reply CHANGELOG.md:6]: I figured it out :)

**Review by armab — APPROVED** [review]
Thanks for catching the bug and for the fix! :+1:

**Pull request merged**